src/app/pages/network/network.component.ts

```diff
@@ -242,21 +247,12 @@ export class NetworkComponent implements OnInit, OnDestroy {
       });
 
     if (this.systemGeneralService.getProductType() === ProductType.ScaleEnterprise) {
-      this.ws
-        .call('failover.licensed')
-        .pipe(untilDestroyed(this))
-        .subscribe((isHaLicensed) => {
-          if (isHaLicensed) {
-            this.ws
-              .call('failover.disabled.reasons')
-              .pipe(untilDestroyed(this))
-              .subscribe((reasons) => {
-                if (reasons.length === 0) {
-                  this.isHaEnabled = true;
-                }
-              });
-          }
-        });
+      this.store$.select(selectIsHaLicensed).pipe(untilDestroyed(this)).subscribe((isHaLicensed) => {
+        if (isHaLicensed) {
+          this.store$.dispatch(loadHaStatus());
```

Comment:
Now that everything is one store and is already loaded on admin ui initialization, dispatch (and conditions) seem unnecessary.